Core/Core/ViewModifiers/TestIdentifier.swift

```diff
@@ -137,4 +137,8 @@ extension View {
         self
     }
     #endif
+
+    public func identifier(_ id: String) -> some View {
+        accessibility(identifier: id).testID(id)
```

Comment:
This would be more useful if testID preceded a11y id, because testID records the type of the view it immediately wraps.

Reply:
Don't we want to capture as much as possible, putting the testID very last?

Reply:
In theory, if testID were able to capture more information, but instead it will just get a type of something like `SwiftUI._AccessibilityModifier<ActualViewWeCareAbout>`

Reply:
In a later PR I'd like to see if I can capture more info automatically from testID.